matrix_client/api.py

```diff
@@ -238,6 +238,16 @@ def send_message(self, room_id, text_content, msgtype="m.text"):
             self.get_text_body(text_content, msgtype)
         )
 
+    def send_html(self, room_id, html_content, msgtype="m.text"):
+        """Send an html formatted message.
+
+        Args:
+            room_id (str): The room ID to send the event in.
+            html_content (str): The html formatted message to be sent.
+        """
+        return self.send_message_event(room_id, "m.room.message",
+                                       self.get_html_body(html_content, msgtype))
```

Comment:
I'd like to see ``self.get_html_body`` moved into room in all honestly. This seems to be a function that has lingered here a while, but I don't see why we are processing html/content in api.py. 

I'd prefer to have ``send_html`` in room.py have an optional parameter for body which would either be filled in by the developer, or generated by our little function here. The content object should be assembled in room.py and sent along to api.py.